src/index.css

```diff
@@ -56,15 +56,31 @@ button:focus-visible {
   outline: 4px auto -webkit-focus-ring-color;
 }
 
-@media (prefers-color-scheme: light) {
-  :root {
-    color: #213547;
-    background-color: #ffffff;
-  }
-  a:hover {
-    color: #747bff;
-  }
-  button {
-    background-color: #f9f9f9;
-  }
+.l-section {
+  max-width: 100vw;
+  text-align: center;
+}
+
+.l-section--home{
+  display: flex;
+  justify-content: center;
+  padding: 20px;
+}
```

Comment:
maybe let's also add `flex-direction: column` here? so that the flex items are placed below the previous one, instead of next to each other